### ISSUE TITLE
Log when files begin to download

### ISSUE THREAD

**Issue by tysg** [fluminurs/fluminurs#630]
The CLI tool currently doesn't log anything when a file starts to download. If the file being downloaded is very large, it will appear to the user that the program is not responding/hanged. It will be nice to show a little message that the files are indeed downloading.

Would also be nice to add a progress bar or something as a better indication.

**Comment by angelsl** [comment]
Yeah, I'm thinking of adding that, after I revamp the error handling.

We can look at https://github.com/mitsuhiko/indicatif or https://github.com/a8m/pb.
